Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -30,16 +30,19 @@ configuration:
   name: template
   required: false
   type: 0
-- display: Use URL Filtering for auto enrichment
+- additionalinfo: "add here"
```

Comment:
If selected, when running the !url command, the command will execute using pan-os with PAN_DB (with applied filters). The URL filtering categories determine DBot score (malicious, suspicious, benign).